modules/installation-launching-installer.adoc

```diff
@@ -159,6 +159,8 @@ You can run the `create cluster` command of the installation program only once,
 
 ifndef::osp,rhv,vsphere[* Configure an account with the cloud platform that hosts your cluster.]
 
+ifdef::rhv[* Open the `ovirt-imageio` port from the machine running the installer to the oVirt Engine. By default, port `54322`.]
```

Comment:
nitpicky stuff...
* Do you mind changing to "...the machine that is running the installer..."?
* I asked in Slack if we can locate the customer-facing name for "oVirt Engine."
* Do you mind using a complete sentence at the end? "By default, the port is `...`."
* Make sure 54322 is the port that you want.  I skimmed the BZ and saw 54323 mentioned.  FYI, I have no technical experience in this area.

Reply:
@mikemckiernan oVirt Engine is upstream term. I actually opened a new bug about this. In RHV we use an attribute as a stand-in for this term. In the upstream it evaluates to "Engine" and in the downstream it evaluates to "Manager".

There is currently no such attribute in the OCP attributes file, but I would like to add one. For now, @ehvs I suggest changing "the oVirt Engine" to "the engine".